docs/source/06_solver_hooks.rst

```diff
@@ -154,6 +154,17 @@ implementing the following Solver `Hooks`.Otherwise, the particle size will be c
 
 .. autofunction:: alfasim_sdk.hook_specs.update_particle_diameter_of_solids_fields
 
+Internal deposit layer
+----------------------
+
+When a new phase is added to the hydrodynamic model using the :class:`~alfasim_sdk.types.AddPhase` type, it is possible
```

Comment:
Why the docs are duplicated, but sightly different, in those two paragraphs?

Reply:
I don't know, maybe it was a merge or a pull with conflict that I didn't see it. I will delete one of them

Reply:
- [ ] Remove duplicate paragraph